content/en/service_management/workflows/connections.md

```diff
@@ -105,6 +106,71 @@ To learn how to restrict connection use, see [Access and Authentication][4].
 
 To connect to an arbitrary service, use the HTTP connection type. For authentication options and setup instructions, see [HTTP action][10].
 
+## Connection identifier tags
+
+You can add identifier tags to connections. The tagging rules for connections are based on [Datadog tags][13], with the following additional requirements:
+- Identifier tags must follow the format `tag:value`, and additional colons are not allowed. For example, the identifier tags `env:staging:east` and `env` are invalid formats for connection tags.
```

Comment:
```suggestion
- Identifier tags must follow the format `<TAG>:<VALUE>`, and additional colons are not allowed. For example, the identifier tags `env:staging:east` and `env` are invalid formats for connection tags.
```
I actually think you should reject this suggestion, it's technically our style but it obscures things unnecessarily here when there's no potential for confusion anyway IMO. But providing it in case you want to do it by the book!

Reply:
Acknowledged :)